### PR TITLE
Fix featureflags not working when /decide is down

### PR DESCRIPTION
**Pull request by timgl**
## Changes

Don't override feature flags when /decide errors, just use the ones still in persistence.

## Checklist
- [ ] Tests for new code (see [advice on the tests we use](https://github.com/PostHog/posthog-js#tiers-of-testing))
- [ ] Accounted for the impact of any changes across different browsers
- [ ] TypeScript definitions (module.d.ts) updated and in sync with library exports (if applicable)

### CODE REVIEW

File: src/decide.js [hunk 1]
```diff
@@ -20,13 +20,16 @@ export class Decide {
...
+            { data: encoded_data, verbose: true },
             { method: 'POST' },
             (response) => this.parseDecideResponse(response)
         )
     }
 
     parseDecideResponse(response) {
+        if (response?.status === 0) {
+            return
+        }
```
Comment (mariusandra) [inline src/decide.js:32]: What about adding logging something here? The console.log statements in this repo are anyway only be visible when `debug:true` is set.

```suggestion
        if (response?.status === 0) {
            console.error('Failed to fetch feature flags from PostHog.')
            return
        }
```

**Review by mariusandra — APPROVED** [review]
I guess that's a fine escape hatch, but I'd still log something to the user when this happens.

**Comment by liyiy** [comment]
Will this fix https://posthog.slack.com/archives/C02G72Y6E9H/p1643293032060800 ?

**Comment by timgl** [comment]
@liyiy Nope but have created #360

**Pull request merged**